src/__tests__/__components__/Messages/ErrorMessage.test.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { shallow } from "enzyme";
+import ErrorMessage from "../../../components/common/Message/error";
+
+describe("Loading", () => {
```

Comment:
@niomwungeri-fabrice Are these tests related to the loading component? Why did you name the describe as `Loading`?